moped-editor/src/utils/mapHelpers.js

```diff
@@ -2,23 +2,53 @@ import React from "react";
 import theme from "../theme/index";
 import { isEqual } from "lodash";
 
-export const MAPBOX_TOKEN = `pk.eyJ1Ijoiam9obmNsYXJ5IiwiYSI6ImNraWV4dHR0ZjAwNnYyd3FwYjFoNHduaDcifQ.--3vRm2KHq1gh5K_L0pqtA`;
+export const MAPBOX_TOKEN = process.env.REACT_APP_MAPBOX_TOKEN;
 
 export const mapInit = {
   latitude: 30.268039,
   longitude: -97.742828,
   zoom: 12,
```

Comment:
I think of the summary page, the map should be more zoomed in by default. 12 seems like a good zoom level for the project extent step in create new project. But when we're talking about a specific existing project, more detail is probably better.

Reply:
This is a great idea! 💡 I created an issue to auto-zoom to the features on the map. We did something similar in the Signs & Markings React map so it shouldn't be too complex. 🤞 https://github.com/cityofaustin/atd-data-tech/issues/4819